doc/build.md

```diff
@@ -10,9 +11,11 @@ You will need the following tools and components to build the Basketweave-S (not
 - 5 PCB mount 2u stabilizers
```

Comment:
4 stabs if split backspace, 69 swicthes if using both encoders

Reply:
Absolutely right thanks

Reply:
Also one more diode in case of split backspace